content/docs/enterprise/branding/error_details.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Error Details
+description: You can show additional Error Details of the 403 Unauthorized Route Pages
+---
+
+It is possible to customize the error messages displayed on 403 unauthorized pages.
+
+This feature is enabled on a per Route basis by toggling the setting in the General Tab of the Route settings.
+
+![Enable Error Details](img/enable_error_details.png)
```

Comment:
Not a grammar issue but it's not immediately clear to my eye what toggle needs to happen. Is it the "Show Error Details" toggle? 

My suggestion is use this same picture but with an arrow pointing to the toggle.